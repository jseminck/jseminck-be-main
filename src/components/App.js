import React, { PropTypes } from 'react';
import Navigation from './Navigation';

const App = (props) => {
  return (
    <div>
      <Navigation />
      {props.children}
    </div>
  );
};

App.propTypes = {
  children: PropTypes.element
};

export default App;
