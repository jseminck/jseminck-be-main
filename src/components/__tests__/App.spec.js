import React from "react";
import renderer from 'react-test-renderer';
import App from "./../App";

describe("App", function() {
    it('renders correctly', () => {
        const app = renderer.create(
          <App>
              <div>
                  Hello World
              </div>
          </App>
        ).toJSON();

        expect(app).toMatchSnapshot();
    });
});