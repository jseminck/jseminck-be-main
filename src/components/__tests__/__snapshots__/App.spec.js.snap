exports[`App renders correctly 1`] = `
<div>
  <div
    className="container">
    <a
      className="item"
      onClick={[Function bound handleClick]}
      style={Object {}}>
      Home
    </a>
     | 
    <a
      className="item"
      onClick={[Function bound handleClick]}
      style={Object {}}>
      Servers
    </a>
     | 
    <a
      className="item"
      onClick={[Function bound handleClick]}
      style={Object {}}>
      Login
    </a>
  </div>
  <div>
    Hello World
  </div>
</div>
`;
